packages/select/src/components/suggest/suggest.tsx

```diff
@@ -182,7 +182,7 @@ export class Suggest<T> extends AbstractPureComponent<SuggestProps<T>, SuggestSt
         if (this.state.isOpen === false && prevState.isOpen === true) {
             // just closed, likely by keyboard interaction
             // wait until the transition ends so there isn't a flash of content in the popover
-            const timeout = this.props.popoverProps?.transitionDuration ?? Popover.defaultProps.transitionDuration;
+            const timeout = this.props.popoverProps?.transitionDuration ?? 300;
```

Comment:
nitpick: Do we want to use a `POPOVER_DEFAULT_PROPS` export here as well?

Reply:
Hm, I'm more inclined to revert my original change here since Popover is a class component (and thus should be exempt from the deprecation). aefacb1fa9115485bebaf01e5cb0b2e0726223a7

Reply:
That works too!